lib/api/src/grpc/qdrant.rs

```diff
@@ -2397,6 +2434,29 @@ impl WriteOrderingType {
 }
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
 #[repr(i32)]
+pub enum ReadConsistencyType {
+    /// Send request to all nodes and return points which present on all of them
```

Comment:
```suggestion
    /// Send request to all nodes and return points which are present on all of them
```

same suggestion on all variants :)